### ISSUE TITLE
Sonobus Android on Oneplus phone touch position versus LCD icon position is off:

### ISSUE THREAD

**Issue by tandefelt** [sonosaurus/sonobus#150]
Hi,

**Thank you for the amazing software, and for the creative/collaborative possibilities it offers.**
I am part of an artist team considering to use Sonobus on a mobile sound art installation with multiple 5G/Android phone based remote recording stereo/binaural stations in the nature in Helsinki Finland (remote accessed and controlled with google remote desktop, Rode AI Micro based A/D converter). 

**Android LCD Display / Touch Screen scaling bug:**
In Android testing, a bug appeared at least on Oneplus 9 phone, Qualcomm 888 based relatively powerful Android phone:
- Initially, touching icons on top and bottom of the screen did not appear to work at all, 
- But, ultimately, we figured out that.. the touch positions versus the screen icon positions had an offset somehow, approx 1cm, on top and bottom of the screen area, whlie, the center of the screen things appeared to work.

Hope this helps in sorting out the issue, a bit challenging to use without the proper syncing of the icons and their touch areas.

Best regards,

Marko.

**Comment by essej** [comment]
Yep, I know what's going on here. I'm working on an update of the android version (which hasn't been updated in a looong time) to solve that, plus lets file recording work on recent Android versions.

**Comment by tandefelt** [comment]
Hi, 
Thank you so much for your reply, sounds great. Looking forward to.. !
best regards, Marko. 
1952 Olympic Village, Helsinki, Finland.

**Comment by essej** [comment]
Should be fixed now in the latest released version 1.6.1...
